cookbooks/concourse/recipes/default.rb

```diff
@@ -0,0 +1,105 @@
+package 'linux-generic-lts-vivid'
+package 'postgresql'
+
+db_password = shell_out("openssl rand -base64 32").stdout.strip
+
+concourse_binary_release = shell_out("curl https://api.github.com/repos/concourse/concourse/releases | grep browser_download_url | grep 'linux_amd64' | head -n 1 | cut -d'\"' -f4").stdout.strip
+
+service 'concourse-web' do
+  action [:stop]
+  only_if { File.exist?("/etc/init/concourse-web.conf") }
+end
+
+execute 'drop-atc-database' do
+  command "psql -c 'DROP DATABASE IF EXISTS atc'"
+  user "postgres"
+  action :run
+end
+
+execute 'drop-atc-user' do
+  command "psql -c 'DROP ROLE IF EXISTS atc;'"
+  user "postgres"
+  action :run
+end
+
+execute 'create-postgres-user' do
```

Comment:
I went with basic `shell_out` / `execute` blocks to configure postgres rather than depending on the `database` / `postgresql` cookbooks to keep our exposure to the chef ecosystem limited.

Switching to using the cookbooks to manage our database would require us to either vendor all the dependencies those cookbooks have, or deal with adding Berkshelf to our build toolchain neither of which I thought were good solutions given how simple our configuration is.